README.md

```diff
@@ -0,0 +1,13 @@
+# Modding Standards & Utilities
+is a modding library for the game [Battle Brothers](https://store.steampowered.com/app/365360/Battle_Brothers/)
+allowing for better mod compatibility
+and providing commong functions for mods.
+
```

Comment:
I would add some more flashy info e.g. "provides groundbreaking modding features that significantly enhance mod compatibility and allows modders to write faster, cleaner, and more sustainable code" etc.

Reply:
I wouldn't wanna lay it on _that_ thick, but I see where you're coming from.

Reply:
Actually I take it back, I think it's fine. I think we can flatter ourselves in the nexusmods description where the users can see it, this is meant to be fairly minimalistic imo (at least for now).

Reply:
Yeah, as long as MSU comes across as something important. I'd still add some adjective like "significantly better" rather than just "better".